ext/dapr-ext-fastapi/dapr/ext/fastapi/app.py

```diff
@@ -24,13 +27,16 @@ class DaprApp:
         app_instance: The FastAPI instance to wrap.
     """
 
-    def __init__(self, app_instance: FastAPI):
+    def __init__(self, app_instance: FastAPI,
+                 router_tags: Optional[List[str]] = DEFAULT_ROUTER_TAGS):

```

Comment:
i don't recommend using the default value like this
`DEFAULT_ROUTER_TAGS` is a list and its immutable which can cause some miss understanding
For example if you will have the next code it can cause some issues
```
a = []
def c(b=a):
    return b
 
 print(a())   # []
 a().append(1)
 print(a()) # [1]
 ```
 first call will have an empty list and on the next one we will append value to the result list which will cause the next runs to have that value as well

Reply:
this comment applies to actor.py as well

Reply:
This is a great comment and I tried the code above and it would be a concern if router_tags would to be returned to the caller. On the other hand, it is kept private within the SDK code.

To avoid this, we could copy-and-paste `['xyz']` into each method but it would hurt readability IMO.

Reply:
I would recommend name the default as private with _ to communicate these should not be touched.

Reply:
My final offer 😄 

```python
    def __init__(self, app_instance: FastAPI,
                 router_tags: Optional[List[str]] = ['PubSub']):
```

Reply:
By the way in your example @yairsimantov20 `a` was not intended to be mutable. It was a constant. Anyone who wanted to use a different value for `a` (`router_tags`) needs to provide it in the constructor as a keyword argument!